agent/src/test/java/com/navercorp/pinpoint/plugin/hystrix/HystrixTestConstants.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2014 NAVER Corp.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.navercorp.pinpoint.plugin.hystrix;
+
+/**
+ * @author Jiaqi Feng
+ *
+ */
+public interface HystrixTestConstants {
+    public static final String VERSION = "1.5.2-SNAPSHOT";
```

Comment:
Not sure about this but should this be `1.5.2-SNAPSHOT`?
You could also use [Version.java](https://github.com/naver/pinpoint/blob/master/commons/src/main/java/com/navercorp/pinpoint/common/Version.java) if you have pinpoint-commons dependency.